main/accounts/index.ts

```diff
@@ -115,6 +117,10 @@ export class Accounts extends EventEmitter {
 
   rename (id: string, name: string) {
     this.accounts[id].rename(name)
+    const account = this.accounts[id].summary()
+    this.update(account)
+    const accountMetaId = uuidv5(id, accountNS)
+    store.updateAccountMeta(accountMetaId, account)
```

Comment:
we might want to consider moving this into the store action that updates an account for two reasons:
1. it will ensure that the names are always updated in tandem and that we don't miss updating one or the other
2. it will remove the need for us to do both of these actions everywhere we update the name